runtime/drivers/registry.go

```diff
@@ -44,6 +42,17 @@ type Instance struct {
 	IngestionLimitBytes int64 `db:"ingestion_limit_bytes"`
 	// Annotations to enrich activity events (like usage tracking)
 	Annotations map[string]string
+	// Instance specific connectors
+	Connectors []*runtimev1.ConnectorDef
+	// RillYAML is the project's parsed rill yaml
+	// Its just a cache and can be reproduced from rill.yaml
+	// TODO :: deprecate ProjectVariables once completely moved to v1 parsing
+	RillYAML *RillYAML `db:"rill_yaml"`
```

Comment:
Instead of deprecating `ProjectVariables`, I think we should add `ProjectConnectors` here as well, but not include a `RillYAML` fields (which is just a parser concern, not part of the internal representation)

Reply:
Instead of adding separate `ProjectVariables` and `ProjectConnectors` field it seems better to just add a single field with all project details as cache. I can rename `RillYAML` to something more generic like `ProjectDetails`/`ProjectInfo` or even `Project`.
Although I think `RillYAML` itself isn't too bad either since we are calling it just a cache of parsed rill yaml.

Reply:
I understand your point, I think though it feels simpler to just store it flattened (especially since it's not removing `ProjectVariables` now). It's also weird to call it `RillYAML` if it only has the variables and connectors, not the other fields in `rill.yaml`

Reply:
I think this is the classical discussion of `flatten` vs `nested`. Flatten is simple, nested is extensible. 
I can flatten it for now but once we completely move to v1 parsing and extend rill yaml further, it may be a good opportunity to nest all these fields.